### PR TITLE
Remove deprecated code using the old preferences system

### PR DESCRIPTION
**Pull request by sprintr**
Follow up of #12182

### CODE REVIEW

**Comment by zaggino** [comment]
To be merged for 1.9, we're nearing 1.8 release (I hope) so it'd be better to merge after release. Methods have been deprecated for over a year so it's time to remove them.

**Comment by le717** [comment]
1.8 has been released and we are currently in 1.9. Time to merge this, maybe? :D

**Comment by ficristo** [comment]
@le717 is there any difference with your PR?

**Comment by le717** [comment]
@ficristo Other than this one being up-to-date and me not having the Brackets source on my computer, probably not. Just merge this one and credit @sprintr. It's cool. :)

**Comment by ficristo** [comment]
> not having the Brackets source on my computer
😢

**Comment by sprintr** [comment]
Or merge @le717's PR, I don't even have a brackets installation. ;)

**Comment by ficristo** [comment]
> I don't even have a brackets installation

😢

**Review by ficristo — APPROVED** [review]

File: src/preferences/PreferencesManager.js [hunk 1]
```diff
@@ -30,198 +31,16 @@
...
-            });
-        }
-
-        return new OldPreferenceStorage(clientID, prefs);
-    }
-
-    /**
-     * Save all preference clients.
-     */
-    function savePreferences() {
```
Comment (zaggino) [inline src/preferences/PreferencesManager.js]: this method is being referenced for example here: https://github.com/adobe/brackets/blob/master/src/document/DocumentCommandHandlers.js#L1378

but also on multiple places

**Review by zaggino — COMMENTED** [review]
this PR seems to delete too much, are we 100% sure it only deletes unused stuff?

File: src/preferences/PreferencesManager.js (hunk 1 above)
Comment (sprintr) [inline src/preferences/PreferencesManager.js]: Yes, savePreferences has been in the deprecated code.

**Comment by zaggino** [comment]
Merged, gives us time to catch problems before next release if they arise.

**Pull request merged**